test/new-e2e/system-probe/system-probe-test-env.go

```diff
@@ -90,84 +109,103 @@ func NewTestEnv(name, securityGroups, subnets, x86InstanceType, armInstanceType
 		"microvm:arm64AmiID":                     auto.ConfigValue{Value: opts.ArmAmiID},
 		"microvm:workingDir":                     auto.ConfigValue{Value: CustomAMIWorkingDir},
 	}
+	// We cannot add defaultPrivateKeyPath if the key is in ssh-agent, otherwise passphrase is needed
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
You can skip passphrase on local environment, on macOs, registering the key with apple key chain

```bash
ssh-add --apple-use-keychain <path/to/private_key>
```